app/views/lookup/new.html.erb

```diff
@@ -1,10 +1,12 @@
 <%= display_header title: "Tag content", breadcrumbs: ["Tag content"] %>
 
-<%= form_for @lookup, url: find_by_slug_path do |f| %>
+<%= simple_form_for @lookup, url: find_by_slug_path do |f| %>
   <div class="form-group">
-    <%= f.text_field :base_path,
-      class: 'form-control',
-      label: "Path or URL of a page on GOV.UK (eg. /book-driving-test)" %>
+    <%= f.input :base_path,
+      label: "Path or URL of a page on GOV.UK (eg. /book-driving-test)",
```

Comment:
The label appears for me:

<img width="761" alt="screen shot 2016-08-09 at 11 47 51" src="https://cloud.githubusercontent.com/assets/136777/17513871/7d79ed62-5e27-11e6-9a35-3cde332880c8.png">